tests/fixtures/testModel.ts

```diff
@@ -440,9 +440,31 @@ export interface GenericModel<T> {
   result: T;
   union?: T | string;
   nested?: GenericRequest<T>;
+  heritageCheck?: ThingContainerWithTitle<T>;
```

Comment:
@WoH I don't necessarily understand what's happening in this PR, but I don't see anything dangerous. So I'm leaning towards "rubber stamping it" (as some people refer to PRs that don't get a lot of scrutiny). But can you clarify what you mean by "heritage?"

Reply:
it's the TS naming. I'm scanning the heritage clauses (extends ...), which is a Typescript AST Node.
I'm ready to rename it though, not sure what a better name would be though.